src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/Models/ElasticPool.cs

```diff
@@ -0,0 +1,117 @@
+        /// </summary>
+        public ElasticPool() { }
+
+        /// <summary>
+        /// Initializes a new instance of the ElasticPool class.
+        /// </summary>
+        /// <param name="location">Resource location</param>
+        /// <param name="name">Resource name</param>
+        /// <param name="id">Resource Id</param>
+        /// <param name="type">Resource type</param>
+        /// <param name="tags">Resource tags</param>
+        /// <param name="creationDate">The creation date of the Azure SQL
+        /// Elastic Pool (ISO8601 format).</param>
+        /// <param name="state">The state of the Azure SQL Elastic Pool.
+        /// Possible values include: 'Creating', 'Ready', 'Disabled'</param>
+        /// <param name="edition">The edition of the Azure SQL Elastic Pool.
+        /// Possible values include: 'Basic', 'Standard', 'Premium'</param>
+        /// <param name="dtu">The total shared DTU for the SQL Azure Database
+        /// Elastic Pool.</param>
+        /// <param name="databaseDtuMax">The maximum DTU any one SQL Azure
+        /// Database can consume.</param>
+        /// <param name="databaseDtuMin">The minimum DTU all SQL Azure
+        /// Databases are guaranteed.</param>
+        /// <param name="storageMB">Gets storage limit for the SQL Azure
+        /// Database Elastic Pool in MB.</param>
+        public ElasticPool(string location, string name = default(string), string id = default(string), string type = default(string), System.Collections.Generic.IDictionary<string, string> tags = default(System.Collections.Generic.IDictionary<string, string>), System.DateTime? creationDate = default(System.DateTime?), string state = default(string), string edition = default(string), int? dtu = default(int?), int? databaseDtuMax = default(int?), int? databaseDtuMin = default(int?), int? storageMB = default(int?))
+            : base(location, name, id, type, tags)
+        {
+            CreationDate = creationDate;
+            State = state;
+            Edition = edition;
+            Dtu = dtu;
+            DatabaseDtuMax = databaseDtuMax;
+            DatabaseDtuMin = databaseDtuMin;
+            StorageMB = storageMB;
+        }
+
+        /// <summary>
+        /// Gets the creation date of the Azure SQL Elastic Pool (ISO8601
+        /// format).
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.creationDate")]
+        public System.DateTime? CreationDate { get; private set; }
+
+        /// <summary>
+        /// Gets the state of the Azure SQL Elastic Pool. Possible values
+        /// include: 'Creating', 'Ready', 'Disabled'
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "properties.state")]
+        public string State { get; private set; }
+
+        /// <summary>
+        /// Gets or sets the edition of the Azure SQL Elastic Pool. Possible
+        /// values include: 'Basic', 'Standard', 'Premium'
+        /// </summary>
```

Comment:
This is generated from the swagger, so I will address this in a separate PR for adding the capabilities api swagger.